assets/js/googlesitekit/widgets/datastore/contexts.test.js

```diff
@@ -0,0 +1,126 @@
+/**
+ * `core/widgets` data store: widget context tests.
+ *
+ * Site Kit by Google, Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import { createTestRegistry } from '../../../../../tests/js/utils';
+import { CORE_WIDGETS } from './constants';
+import Null from '../../../components/Null';
+
+describe( 'core/widgets Widget context selectors', () => {
+	let registry;
+
+	beforeEach( () => {
+		registry = createTestRegistry();
+	} );
+
+	describe( 'isWidgetContextActive', () => {
```

Comment:
Forgot to mention this, but we need to update the structure of this test file to align with other datastore-related tests. The top describe block should define the partial datastore name, then it should have a child describe block for selectors which, in turn, should have its own describe block for an individual selector:

```js
describe( 'core/widgets contexts', () => {
    let registry;

    beforeEach( () => {
        registry = createTestRegistry();
    } );
    
    describe( 'selectors', () => {
        describe( 'isWidgetContextActive', () => {
            ...
        } );
    } );
} );
```

Reply:
No worries. I had actually started off with this structure but had removed the "extra" describe, I thought from previous comments it might be extraneous. :laughing: All good, I've pushed this change, thanks.